xml/deployment_hwrecommend.xml

```diff
@@ -33,27 +33,28 @@
   <itemizedlist>
    <listitem>
     <para>
-     A trusted internal network, the back-end network called the
-     <literal>cluster network</literal>.
+     A front-end network called the <literal>public network</literal>.
     </para>
    </listitem>
    <listitem>
     <para>
-     A public client network called <literal>public network</literal>.
+     A trusted internal network, the back-end network called the
+     <literal>cluster network</literal>. This is optional.
     </para>
    </listitem>
    <listitem>
     <para>
-     Client networks for gateways, these are optional.
+     One or more client networks for gateways. This is optional and
+     beyond the scope of this chapter.
     </para>
    </listitem>
   </itemizedlist>
 
```

Comment:
Here we are missing a paragraph describing the public network. Maybe this one:

"The public network is the network over which Ceph daemons communicate with each other and with their clients. That means: all Ceph cluster traffic goes over this network except in the case when a cluster network is configured (see the next paragraph)."

Reply:
Depends. Are we assuming the user doesn't understand what a public network is vs. a cluster network? In that case, we should probably add them as glossary definition and cross-ref that, rather than insert definitions :thinking:

Reply:
Isn't is weird that there is a paragraph describing what the cluster network is, then? Either have paragraphs describing both public and cluster, or no paragraphs describing either.

Reply:
It is, to be honest. It stands out like a sore thumb. What I meant by my comment above is that the cluster network definition is a standout, and an awkward one. We can define both, but my recommendation would be for both to go into the glossary. What are your thoughts, @tbazant ?

Reply:
After reviewing this again, decided to go with both.